src/pages/AppProfile.vue

```diff
@@ -240,14 +244,19 @@ export default {
     email.value = store.getters.userInfo.email;
     const router = useRouter();
 
+    // Need this to maintain state of organisation details before editing
+    // onCancel we can revert organisationDetails to this state and onSave we can update this state
+    let organisationDetailsActualState = {};
```

Comment:
Consider removing the comments and renaming the variable to `organisationDetailsResetState` to clarify the intent.

```suggestion
    let organisationDetailsResetState = {};
```

Reply:
Done